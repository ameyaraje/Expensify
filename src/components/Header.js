import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => {
    return (
        <header>
            <h1>Expensify App</h1>
            <NavLink to="/" activeClassName="is-active" exact>Home</NavLink>
            <NavLink to="/add" activeClassName="is-active" exact>Add Expense</NavLink>
            <NavLink to="/help" activeClassName="is-active" exact>Help Page</NavLink>
        </header>
    );
};

export default Header;