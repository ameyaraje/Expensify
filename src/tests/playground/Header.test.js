import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';

import Header from '../../components/Header';

test('should render Header', () => {
    const renderer = new ReactShallowRenderer();

    renderer.render(<Header />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
    
});
