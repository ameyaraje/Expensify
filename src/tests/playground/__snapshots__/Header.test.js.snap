// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header 1`] = `
<header>
  <h1>
    Expensify App
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/add"
  >
    Add Expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/help"
  >
    Help Page
  </NavLink>
</header>
`;
